packages/react/src/primitives/types/message.ts

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+
+import { BaseFlexProps } from './flex';
+import { BaseButtonProps } from './button';
+import { ElementType, PrimitiveProps, BaseViewProps } from './view';
+
+export type MessageColorTheme =
```

Comment:
This probably doesn't need to a public export right? Only external usage seems to be for `MessageContextProps`